WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -56,19 +62,21 @@
     private int mSpinnerDrawableRight;
     private AppLog.T mTAG;
 
+    private boolean mToolbarVisibilityLock = false;
```

Comment:
Can we rename `mToolbarVisibilityLock` to something that more closely matches its purpose, like `mHideToolbarOnScroll` perhaps?

Reply:
if true, this variable locks the toolbar so it cannot be hidden - so `mHideToolbarOnScroll` would be misleading. I think this confusion comes from the fact that I had it initialized to false here, but then during init - I default to true, which is a bug that I noticed when looking at your other comment 😕

Reply:
Well, the idea was to name the variable to better describe what it does since `mToolbarVisibilityLock` doesn't let future devs know it means "don't hide the toolbar when scrolling." I'm fine with renaming it to anything that's more descriptive.

Reply:
Ok. I've renamed the styleable attribute to `wpToolbarDisableScrollGestures` and the variable to `mToolbarDisableScrollGestures`.